.pylintrc

```diff
@@ -423,4 +423,4 @@ analyse-fallback-blocks=no
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception"
-overgeneral-exceptions=Exception
+overgeneral-exceptions=builtins.Exception
```

Comment:
why did this have to change if the disable pylint rule is added?

Reply:
i was getting a warning for just using `Exception`. pylint wated `builtins.Exception`